helm-charts/splunk-otel-collector/templates/securityContextConstraints.yaml

```diff
@@ -41,6 +41,7 @@ kind: SecurityContextConstraints
 apiVersion: security.openshift.io/v1
 metadata:
   name: {{ template "splunk-otel-collector.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
I think this resource is also cluster-scoped. @jvoravong can you confirm if `SecurityContextConstraints` in OC is cluster or namespace scoped?

Reply:
[Cluster scoped](https://docs.openshift.com/container-platform/4.15/authentication/managing-security-context-constraints.html#role-based-access-to-ssc_configuring-internal-oauth).

Reply:
Also removed